entry_types/scrolled/package/src/entryState/structure.js

```diff
@@ -105,12 +103,29 @@ export function useSectionContentElements({sectionId}) {
 
 export function useChapters() {
   const chapters = useEntryStateCollectionItems('chapters');
-
+  let chapterTitles = {};
   return chapters.map(chapter => {
+    let chapterRef = chapter.configuration.title;
+    if (chapterRef) {
+      if (slugify) {
```

Comment:
**question**: When would this be false?

Reply:
While testing out different libraries, initial ones were undefined during server side rendering.
But slugify is not undefined so it was unnecessary.